Add Python 3.11 (test), Modify package combination

milljm opened the pull request:
Allow Ptyhon 3.11 packages to become available. TODO: we have no tests for this version yet. Need to create some.

Attempt to align the Paraview package version which gets installed on Linux (5.10) to match with what is being installed on Mac (5.11)

Closes #25498

Commit:
Add Python 3.11 (test), Modify package combination

Allow Ptyhon 3.11 packages to become available. TODO: we have no
tests for this version yet. Need to create some.

Attempt to align the Paraview package version which gets installed
on Linux (5.10) to match with what is being installed on Mac (5.11)

Closes #25498

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1775145/) on edb1f93 wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1775145/) on edb1f93 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/1775169/) on edb1f93 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/25499/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/1775187/) on edb1f93 wanted to post the following:


## Framework coverage

Coverage did not change


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/geochemistry)
- [`heat_conduction`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/heat_conduction)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/navier_stokes)
- [`optimization`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/optimization)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/richards)
- [`scalar_transport`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/scalar_transport)
- [`solid_properties`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/solid_properties)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/25499/coverage/xfem)

</details>



This comment will be updated on new commits.

milljm commented:
I'd like to look into this: #25509 before merging...

Commit:
Cannot ever do it right in one go...

Forgot, or miplaced, or whatever... bump libMesh, and moose-dev.

Commit:
Apply matplotlib 3.8 API change fix

A recent update to matplotlib sees some systax no longer deprecated,
and is causing failures for several tests.

Closes #25509

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1777224/) on 720cba1 wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1777224/) on 720cba1 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1777224/) on 720cba1 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops
